crates/cac_client/src/interface.rs

```diff
@@ -111,22 +110,22 @@ pub extern "C" fn cac_new_client(
             .create_client(tenant.clone(), duration, hostname)
             .await
         {
-            Ok(_) => return 0,
+            Ok(_) => 0,
```

Comment:
This doesn't terminate the function right? That should happen here. Early returns are required